Microsoft.Toolkit.Uwp.SampleApp/SamplePages/SystemInformation/SystemInformationPage.xaml

```diff
@@ -212,7 +212,7 @@
                                TextWrapping="Wrap" />
                     <TextBlock Margin="20,0"
                                FontSize="18"
-                               Text="{x:Bind LastResetTimeProp, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}"
+                               Text="{x:Bind LastResetTime, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}"
```

Comment:
I forgot to ask before, any reason why this is a TwoWay binding instead of just OneWay?  You should just need `{x:Bind LastResetTime, Mode=OneWay}`

Reply:
Yeah. That was mandatory to listen to PropertyChanged event. I will fix that too.